src/components/AppLayout/Sidebar/SafeHeader/index.tsx

```diff
@@ -60,15 +64,63 @@ const StyledIcon = styled(Icon)`
 const IconContainer = styled.div`
   width: 100px;
   display: flex;
-  padding: 4px 0;
+  gap: 8px;
   justify-content: space-evenly;
+  margin: 14px 0;
 `
 const StyledButton = styled(Button)`
   &&.MuiButton-root {
+    width: 100%;
+    height: 38px;
     padding: 0 12px;
   }
-  *:first-child {
-    margin: 0 4px 0 0;
+`
+
+const StyledExplorerButton = styled(ExplorerButton)`
+  border-radius: 5px;
+  width: 32px;
+  height: 32px;
+  background-color: ${background};
+
+  & .icon-color {
+    fill: #008c73;
+  }
+
+  &:hover {
+    background-color: #effaf8;
```

Comment:
It is probably worthwhile adding this to the variables file.

Reply:
Just realized we already have it as `primaryLite` but I also added `primaryGreen200` because of different naming in our style guide. Removed `primaryGreen200` again and use `primaryLite` now everywhere. We can tackle possible renaming of this in a later task.